dev/planning-example.py

```diff
@@ -0,0 +1,236 @@
+               pressure=0.615 * u.bar,
+               relative_humidity=0.11,
+               temperature=0 * u.deg_C,
+               timezone=pytz.timezone('US/Hawaii'),
+               description="Subaru Telescope on Mauna Kea, Hawaii")
+
+# Define the same observer with an instance of astropy.coordinates.EarthLocation,
+# also use pytz.timezone() argument directly as `timezone` keyword's input
+from astropy.coordinates import EarthLocation
+obs = Observer(name='Subaru',
+               location=EarthLocation(lon='-155:28:48.900',
+                                      lat='+19:49:42.600'),
+               elevation=4163 * u.meter,
+               pressure=0.615 * u.bar,
+               relative_humidity=0.11,
+               temperature=0 * u.deg_C,
+               timezone='US/Hawaii',
+               description="Subaru Telescope on Mauna Kea, Hawaii")
+
+# longitude and latitude would be represented internally by astropy
+# coordinates.angles.Longitude/Latitude.  So any value that can naturally 
+# initialize one of those objects could be passed as the parameter to Observer
+
+# It's also possible to initialize using astropy objects and other
+# direct entities
+from astropy.coordinates import Longitude, Latitude
+
+obs = Observer(name='Keck 1',
+               longitude=Longitude(155.4750 * u.degree),
+               latitude=Latitude(19.8264 * u.degree),
+               elevation=4160 * u.m,
+               pressure=0.615 * u.bar,
+               relative_humidity=0.11,
+               temperature=0.0 * u.deg_C,
+               timezone=pytz.timezone('US/Hawaii'),
+               description="W.M. Keck 1 Telescope on Mauna Kea, Hawaii")
+
+# It would also be desirable to be able to have a small database of
+# common telescopes.  Maybe this can at first simply take the form of
+# a python module:
+from astroplan import sites
+
+obs = sites.Keck1
+
+# Environmental conditions should be updatable.  Suggest using keyword
+# parameters (this would make it easy to update an internal AltAz coordinate):
+
+obs.set_environment(pressure=0.600 * u.bar, relative_humidity=0.2,
+                    temperature=10 * u.deg_C)
+
+# Dates are assumed to be UTC by default
+
+"""
+We can ask about typical times of interest at this observing position
+Returned dates are assumed to be in the observer's time zone for convenience.
+The date parameter, if given, can be an astropy Time or a datetime instance.
+If no parameter is given, the current date/time is assumed.
+"""
+
+# Define a time
+from astropy.time import Time
+time_obs = Time(2457189.500000, format='jd')
+
+# sunset
+obs.sunset(date=time_obs, which='nearest') # Default
+obs.sunset(date=time_obs, which='next')
+obs.sunset(date=time_obs, which='previous')
+
+# sunrise
+obs.sunrise(date=time_obs, which='nearest') # Default
+obs.sunrise(date=time_obs, which='next')
+obs.sunrise(date=time_obs, which='previous')
+
+# moon rise
+obs.moon_rise(date=time_obs, which='nearest')
```

Comment:
Agreed. According to Webster, [moonrise/moonset](http://www.merriam-webster.com/dictionary/moonrise) successfully builds in the English language.